specs/OracleQueries.yml

```diff
@@ -1542,7 +1589,39 @@ components:
           type: string
           description: The region
           example: "Surrey"
+    managingOfficerData:
+      type: object
+      required:
+        - residential_address
+        - contact_name_full
+        - contact_email_address
+        - date_of_birth
+      properties:
+        residential_address:
+          type: string
+          description: The residential address of the managing officer
+          example: "123 Updog road"
+        contact_name_full:
+          type: string
+          description: The full contact name of the managing director
+          example: "John doe"
+        contact_email_address:
+          type: string
+          description: The contact email address of the managing officer
+          example: "testing@email.com"
+        date_of_birth:
+          type: string
+          description: The date of birth of the managing officer
+          example: "20-1-1930"
   parameters:
+    incorporationNumber:
+      name: 'incorporation_number'
+      description: The incorporation number
+      in: 'path'
+      required: true
+      schema:
+        type: string
+      example: "12345678"
     companyNumber:
```

Comment:
confused by parameters having both incorporationNumber and (1625) companyNumber - only companyNumber